packages/aws-cdk-lib/aws-lambda/lib/function.ts

```diff
@@ -913,6 +913,12 @@ export class Function extends FunctionBase {
     // add additional managed policies when necessary
     if (props.filesystem) {
       const config = props.filesystem.config;
+      if (!/^\/mnt\/[a-zA-Z0-9-_.]+$/.test(config.localMountPath)) {
+        throw new Error(`Local mount path should match with ^/mnt/[a-zA-Z0-9-_.]+$ but given ${config.localMountPath}.`);
+      }
+      if (config.localMountPath.length > 160) {
+        throw new Error(`Local mount path can not be longer than 160 characters but has ${config.localMountPath.length} characters.`);
+      }
```

Comment:
You need to check if the `LocalMountPath` is a resolved token or not (like a CFN Parameter that will be only available during deployment time)

Reply:
OK, but how?
I am not familiar with CDK internal

Reply:
`Token.isUnresolved(config.localMountPath)`?

Reply:
Changed https://github.com/aws/aws-cdk/pull/31019/commits/bf7911dc4d3bfe1b0b50a91b75aa91cff850e3d2